### ISSUE TITLE
Delete tool_lifecycle backups

### ISSUE THREAD

**Issue by tobiasreischmann** [learnweb/moodle-tool_lifecycle#44]
Currently we store the backups of courses, however, those are never deleted themselves. This could be done either by implementing a delete backup step in combination with a "wait" step or we could add a delete date to every backup stored and delete them accordingly.

**Comment by bfriesenvcc** [comment]
If I understand you correctly, you're asking for an automated process to delete the backups that are made by the lifecycle tool. 

I agree this would be helpful, but I'd also like to be able to manually delete them. At this point it appears that they stay there forever, which is not acceptable for us. Am I missing something?

**Comment by NinaHerrmann** [comment]
Hey @bfriesenvcc,
you are absolutely right, it is more a feature than an issue ;)

**Comment by MSVermet** [comment]
I was wondering if it is already possible to delete the backups that are made by the lifecycle tool? I can only see the options 'download' or 'restore'. How do other users delete old backups?

**Comment by NinaHerrmann** [comment]
No, as we described above, we do not have a process. Currently, we just delete files older than xxxx on the command line. If you are interested in the topic please feel free to submit a proposal for the process!

**Comment by LisaE14** [comment]
@bluetom 
My colleague provided our step yesterday: https://github.com/learnweb/moodle-tool_lifecycle/pull/278

**Comment by LisaE14** [comment]
@bluetom Now that this have been implemented, we should probably close the issue?
